index.html

```diff
@@ -10,11 +10,11 @@
             <li>ls -la: lists hidden flies</li>
         </ul>
     </li>
-    <li>cd</li>
+    <li>cd: change directory</li>
     <li>mkdir: Creates folder</li>
     <li>touch: create file</li>
-    <li>rm -Rf</li>
-    <li>pwd: </li>
+    <li>rm -Rf: remove program</li>
```

Comment:
are we really removing a "program" or is it a directory? In which cases do we remove just a file, and what command do we use to remove a directory?